Rules

```diff
@@ -1,16 +1,18 @@
 #!/usr/bin/env ruby
 
 preprocess do
-  # Add index extra data
-  index = @items['/index.md']
   projects = @items.find_all '/projects/*.md'
   members = @items.find_all '/team/*.md'
+
+  # Add index extra data
+  index = @items['/index.md']
   index[:projects] =  projects
   index[:members] = members
 
   @items.create('', { projects: projects }, '/projects/index.html')
   @items.create('', { members: members }, '/team/index.html')
-  @items.create('', {projects: projects}, '/documentation/index.html')
+  @items.create('', { projects: projects }, '/documentation/index.html')
+  @items.create('', {}, '/repos/index.html')
```

Comment:
Can we remove the preprocessor-created repos index by combining the `repos.erb` layout content into `content/repos/index.html`?

Reply:
Solved in 5bc43c5